lib/CTownHandler.h

```diff
@@ -237,6 +238,7 @@ class DLL_LINKAGE CFaction : public Faction
 		h & index;
 		h & nativeTerrain;
 		h & alignment;
+		h & preferUndergroundPlacement;

```

Comment:
Now you should take care about backward compatibility with older saves. Or do not serialize this flag at all because it is not needed later in game

Reply:
I see the point, just added to this place without full understanding of meaning